portfolio/src/main/webapp/comments-script.js

```diff
@@ -1,3 +1,26 @@
+
+var credentials = {};
+
+function initiateFunctions(){
+    fetch("/auth")
+    .then(res => res.json())
+    .then(res => {
+        credentials = res;
+        if(res.loggedIn === true){
+            document.getElementById("commentForm").style.visibility = "visible";
+            document.getElementById("loginRoute").innerHTML = "Logout";
+            document.getElementById("loginRoute").setAttribute("href", res.logoutUrl);
+        }else{
+            document.getElementById("commentForm").style.visibility = "hidden";
+            document.getElementById("loginRoute").innerHTML = "Login";
+            document.getElementById("loginRoute").setAttribute("href", res.loginUrl);
+        }
+        callFetch(10);
+    })
+    .catch(err => console.error(err));
+}
+
+
 //Retrieve data from the server and create a comment Element for each response.
 function callFetch(value){
     let url = "/data?num="+value.toString();
```

Comment:
Should have pointed this earlier, can you give it a more meaningful name instead of "value"?